templates/zerver/help/add-a-link-preview.md

```diff
@@ -0,0 +1,36 @@
+# Add a link preview
+
+{!admin-only.md!}
+
+Link preview is a way to see an overview of the content before clicking it.
+This can be useful for various reasons, from safety precautions to quick
+integrity check to make sure the user actually posted the right link.
+Here's an example of link preview:
+
+![Link preview example](/static/images/help/link-preview.png)
+
+Follow these steps to enable a link preview:
+
+1. Go to your server's Django settings file, `settings.py`. If you have a
```

Comment:
Do a small check if we have a doc already explaining where the settings file is. In production installs, I think the path was `/etc/zulip/settings.py` or similar. Linking to such a doc would be great.

Reply:
It is briefly mentioned in the [readthedocs](https://zulip.readthedocs.io/en/latest/subsystems/settings.html). Would link it